src/main/java/com/connexta/store/adaptors/FileRetrieveResponse.java

```diff
@@ -13,7 +13,7 @@
 import org.springframework.http.MediaType;
 
 @Data
-public class RetrieveResponse {
+public class FileRetrieveResponse {
```

Comment:
Looking at the fields of FileRetrieveResponse: inputstream, mediatype, and name. This is a model of a File. It is created from a response, but isn't a Response. A good name for the class might be `File`, `IonFile`, or `DataFile`  because that is what it represents.